### PR TITLE
Aedbranch

### PR DESCRIPTION
**Pull request by sujin16**
Merge aedBranch

### CODE REVIEW

**Comment by sujin16** [comment]
Finish aedbranch Merge

**Comment by jieun1114** [comment]
Good job!!

**Comment by sujin16** [comment]
merge 2

File: app/build.gradle [hunk 1]
```diff
@@ -23,20 +23,32 @@ android {
 }
 
 dependencies {
-
```
Comment (sujin16) [inline app/build.gradle]: delete

**Review by sujin16 — COMMENTED** [review]
Update file

**Pull request merged**